src/vaadin-dialog.html

```diff
@@ -289,6 +385,186 @@
             e.preventDefault();
           }
         }
+
+        _modelessChanged(modeless) {
```

Comment:
This observer could be replaced with template bindings to the `<vaadin-dialog-overlay>`:
```
modeless="[[modeless]]"
with-backdrop="[[!modeless]]" (remove the existing with-backdrop attribute)
```

Reply:
Done.